src/improvements/template/TransferOwners.sol

```diff
@@ -0,0 +1,181 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import {ProxyAdmin} from "@eth-optimism-bedrock/src/universal/ProxyAdmin.sol";
+import {VmSafe} from "forge-std/Vm.sol";
+import {stdToml} from "forge-std/StdToml.sol";
+import {LibString} from "solady/utils/LibString.sol";
+import {console} from "forge-std/console.sol";
+
+import {L2TaskBase} from "src/improvements/tasks/types/L2TaskBase.sol";
+import {SuperchainAddressRegistry} from "src/improvements/SuperchainAddressRegistry.sol";
+
+/// @notice Template contract for doing a batch transfer of ownership for a chain.
+/// This includes the L1ProxyAdminOwner, DisputeGameFactory and optionally the Permissioned/Permissionless DelayedWETH contracts.
+/// Some chains may not have a PermissionedWETH or PermissionlessWETH and or may not be ownable. We handle this accordingly.
+/// ATTENTION: Please use caution when using this template. Transferring ownership is high risk.
+contract TransferOwners is L2TaskBase {
+    using stdToml for string;
+    using LibString for string;
+
+    /// @notice OP Mainnet chain info.
+    SuperchainAddressRegistry.ChainInfo internal opMainnetChainInfo =
+        SuperchainAddressRegistry.ChainInfo({chainId: 10, name: "OP Mainnet"});
+
+    /// @notice OP Sepolia chain info.
+    SuperchainAddressRegistry.ChainInfo internal opSepoliaChainInfo =
+        SuperchainAddressRegistry.ChainInfo({chainId: 11155420, name: "OP Sepolia Testnet"});
```

Comment:
not a blocker, but could use `getChain` from `StdChains` to avoid hardcoding these